test/direct.test.js

```diff
@@ -49,5 +49,20 @@ it('supports Safari TP', () => {
 })
 
 it('supports Can I Use cutted versions', () => {
-  expect(browserslist('and_chr 55')).toHaveLength(1)
+  expect(browserslist('and_uc 10')).toHaveLength(1)
+})
+
+it('supports Can I Use missing mobile versions', () => {
+  var opts = { mobileToDesktop: true }
+  expect(browserslist('chromeandroid 53', opts)).toEqual(['and_chr 53'])
+  expect(browserslist('and_ff 60', opts)).toEqual(['and_ff 60'])
+  expect(browserslist('ie_mob 9', opts)).toEqual(['ie_mob 9'])
+  expect(browserslist('op_mob 30', opts)).toEqual(['op_mob 30'])
+})
+
+it('missing mobile versions are not aliased by default', () => {
+  expect(browserslist('chromeandroid 53')).not.toEqual(['and_chr 53'])
+  expect(browserslist('and_ff 60')).not.toEqual(['and_ff 60'])
+  expect(() => browserslist('ie_mob 9')).toThrow()
+  expect(() => browserslist('op_mob 30')).toThrow()
```

Comment:
I don't know why some of these browsers throw and some resolve to another version, but it's the current behavior:
- IE throws: https://browserl.ist/?q=ie_mob+9
- Chrome resolves to a wrong versoin: https://browserl.ist/?q=and_chr+53

Reply:
`browserl.ist` is not official, it uses old Browserslist. Doesn’t use it in investigation.

Reply:
Yeap, error is throwing because Can I Use doesn't have info